tests/ValueParsers/TimeParserTest.php

```diff
@@ -58,372 +55,278 @@ public function validInputProvider() {
 		$valid = array(
 			// Empty options tests
 			'+0000000000002013-07-16T00:00:00Z' => array(
-				TimeValue::newFromArray( array(
-					'time' => '+0000000000002013-07-16T00:00:00Z',
-					'timezone' => 0,
-					'before' => 0,
-					'after' => 0,
-					'precision' => TimeValue::PRECISION_DAY,
-					'calendarmodel' => TimeFormatter::CALENDAR_GREGORIAN
-				) ),
-				$emptyOpts,
+				new TimeValue(
+					'+0000000000002013-07-16T00:00:00Z',
```

Comment:
Having named arguments is nice in that I can tell what those zero's above mean, while below it's a lot less clear. It also opens up the possibility of messing up the order. I personally prefer the current version better.